src/modules/coreHttpApi/controllers/AttributesController.ts

```diff
@@ -74,6 +74,96 @@ export class AttributesController extends BaseController {
         return this.ok(result);
     }
 
+    @GET
+    @Path("/Own/Repository")
+    @Accept("application/json")
+    public async getOwnRepositoryAttributes(@Context context: ServiceContext): Promise<Envelope> {
+        const onlyLatestVersionsQuery = context.request.query.onlyLatestVersions;
+        const onlyLatestVersions: boolean | undefined = typeof onlyLatestVersionsQuery === "string" ? onlyLatestVersionsQuery === "true" : false;
+        const result = await this.consumptionServices.attributes.getRepositoryAttributes({
+            onlyLatestVersions
+        });
+        return this.ok(result);
+    }
+
+    @GET
+    @Path("/Own/Shared/Identity")
+    @Accept("application/json")
+    public async getOwnSharedIdentityAttributes(@Context context: ServiceContext): Promise<Envelope> {
+        const urlQuery = context.request.query;
+
+        const query: Record<string, any> = {};
+
+        Object.entries(urlQuery).forEach(([key, value]) => {
+            if (key.startsWith("query.")) {
+                query[key.replace("query.", "")] = typeof value === "string" ? value : "";
+            }
+        });
+
+        const result = await this.consumptionServices.attributes.getOwnSharedAttributes({
+            peer: typeof urlQuery.peer === "string" ? urlQuery.peer : "",
```

Comment:
`urlQuery.peer ?? ""`

please use a shorthand for these kinds of things

Reply:
same for other occurrences.

Reply:
Sadly, not working as the type of the urlQuery is 

`{ [key: string]: undefined | string | string[] | ParsedQs | ParsedQs[] }`

i will change this to also use the @QueryParam decorator